packages/usehooks-ts/src/useDocumentTitle/useDocumentTitle.ts

```diff
@@ -1,7 +1,17 @@
+import { useRef } from 'react'
+
 import { useIsomorphicLayoutEffect } from '..'
 
-export function useDocumentTitle(title: string): void {
+export function useDocumentTitle(title: string, prevailOnUnmount = true): void {
```

Comment:
We'll use an `options` object parameter to be able to add more params later without breaking the API